src/Integration.Vsix/SonarLint.vsct

```diff
@@ -39,8 +42,23 @@
           <ButtonText>SonarLint</ButtonText>
         </Strings>
       </Menu>
+      <Menu guid="guidCmdSet" id="SonarLintHelpMenuItem" priority="0x9999">
+        <Parent guid="guidCmdSet" id="SonarLintCommandSubMenu"/>
+        <Strings>
+          <ButtonText>Help</ButtonText>
+        </Strings>
+      </Menu>
     </Menus>
 
+    <Buttons>
+      <Button guid="guidCmdSet" id="SonarLintHelpShowLogsCommand">
```

Comment:
Let's discuss the name of commands too, I'd drop the `SonarLint<Category>` prefix and keep it just `ShowLogsCommand`.
(Also open question: Command, or CommandId? Menu vs MenuId?)